src/utils/samlUtils.ts

```diff
@@ -318,9 +325,15 @@ export const getIDFromRequest = (requestXML: string): O.Option<string> => {
 const getAuthnContextValueFromResponse = (
   response: string
 ): O.Option<string> => {
-  const xmlResponse = new DOMParser().parseFromString(response, "text/xml");
+  const maybeXmlResponse = new DOMParser().parseFromString(
+    response,
+    "text/xml"
+  );
+  if (maybeXmlResponse === undefined) {
```

Comment:
maybe we could wrap `new DOMParser().parseFromString` into a new method that returns an `Option` (`None` in case of undefined) and use that one everywhere

Reply:
thank you. solved in https://github.com/pagopa/io-spid-commons/pull/145/commits/87789264a2115666c2e104f4a5f3a053fa5c596f